core/lib/globals_shared.h

```diff
@@ -212,38 +212,60 @@ typedef byte *app_pc;
 typedef void (*generic_func_t)();
 
 #ifdef WINDOWS
-#    ifndef DR_DO_NOT_DEFINE_uint64
-typedef unsigned __int64 uint64;
+#    ifndef DR_DEFINE_FOR_uint64
+#        define DR_DEFINE_FOR_uint64 unsigned __int64
 #    endif
-#    ifndef DR_DO_NOT_DEFINE_int64
-typedef __int64 int64;
+#    ifndef DR_DEFINE_FOR_int64
+#        define DR_DEFINE_FOR_int64 __int64
 #    endif
 #    ifdef X64
 typedef __int64 ssize_t;
 #    else
 typedef int ssize_t;
 #    endif
 #    define _SSIZE_T_DEFINED
-#    define INT64_FORMAT "I64"
+#    ifndef INT64_FORMAT
+#        define INT64_FORMAT "I64"
+#    endif
 #else /* Linux */
 #    ifdef X64
-#        ifndef DR_DO_NOT_DEFINE_uint64
-typedef unsigned long int uint64;
+#        ifndef DR_DEFINE_FOR_uint64
+#            define DR_DEFINE_FOR_uint64 unsigned long int
+#        endif
+#        ifndef DR_DEFINE_FOR_int64
+#            define DR_DEFINE_FOR_int64 long int
 #        endif
-#        ifndef DR_DO_NOT_DEFINE_int64
-typedef long int int64;
+#        ifndef INT64_FORMAT
+#            define INT64_FORMAT "l"
 #        endif
-#        define INT64_FORMAT "l"
 #    else
-#        ifndef DR_DO_NOT_DEFINE_uint64
-typedef unsigned long long int uint64;
+#        ifndef DR_DEFINE_FOR_uint64
+#            define DR_DEFINE_FOR_uint64 unsigned long long int
 #        endif
-#        ifndef DR_DO_NOT_DEFINE_int64
-typedef long long int int64;
+#        ifndef DR_DEFINE_FOR_int64
+#            define DR_DEFINE_FOR_int64 long long int
+#        endif
+#        ifndef INT64_FORMAT
+#            define INT64_FORMAT "ll"
 #        endif
-#        define INT64_FORMAT "ll"
 #    endif
 #endif
+
+#ifdef DR_DEFINE_FOR_uint64
+#undef DR_DO_NOT_DEFINE_uint64
```

Comment:
Looks like a clang-format violation -- prob why Travis failed.  Below too.

Reply:
Done. There was another reason, doxygen got confused by <type>_FORMAT. I fixed by enclosing in quotes, similar to what was done in the previous paragraph.